scripts/zones/Horlais_Peak/mobs/Aries.lua

```diff
@@ -112,16 +105,18 @@ entity.onMobFight = function(mob, target)
     local proximityTimer = 3
 
     handleSleepStates(mob, isAsleep)
-
     -- Use Rumble or Ram Charge twice in a row.
-    if doubleAbility > 0 and not abilityBlocked(mob) then
+    if
+        doubleAbility > 0 and
+        not xi.combat.behavior.isEntityBusy(mob)
+    then
         local abilityID = mob:getLocalVar('doubleAbilityID')
         mob:useMobAbility(abilityID)
     end
 
     -- Use self-sleep after Petribreath.
     if
-        not abilityBlocked(mob) and
+        not xi.combat.behavior.isEntityBusy(mob) and
         not isAsleep and
```

Comment:
This is checking for sleep_I status effect. That is already being checked in the isBusy function